recipes/pyscrap/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pyscrap" %}
+{% set version = "0.0.9" %}
+{% set sha256 = "b15d33ea688381ffccb281bb2a620f13db3a0cb1407544345dca4b80de4e8273" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - import.patch  # [py3k]
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
```

Comment:
Are the dependencies beyond `python` and `setuptools` (or `pip` if you switch over to using that as described in another comment) needed to `build` the package? They seem like runtime-only dependencies, although I'm not sure if `2to3` makes them necessary.